meson.build

```diff
@@ -67,6 +66,16 @@ if openpower_support
     fwupdate_sources += [
         'src/image_openpower.cpp',
     ]
+elif host_image_type == 'intel-x86'
```

Comment:
well, more like `yadro-x86`. Intel's boards don't have this.

Reply:
Also renamed to `intel-c62x`.